Flepper.Tests.Unit/QueryBuilder/Commands/OrderByCommandTests.cs

```diff
@@ -53,19 +52,41 @@ public void ShouldCreateSelectWithWhereAndOrderByStatement()
         }
 
         [Fact]
-        public void ShouldCreateSelectWithWhereAndOrderByDescStatement()
+        public void ShouldCreateSelectBirthdayWithWhereAndMultipleOrderBy()
         {
-            var queryResult = FlepperQueryBuilder.Select("Id", "Name", "Age")
+            var queryResult = FlepperQueryBuilder.Select("Id", "Name", "Birthday")
                 .From("user")
-                .Where("Age").EqualTo(29)
-                .OrderByDesc("Name")
+                .Where("Age").LessThan(29)
+                .OrderBy("Name")
+                .OrderBy("Birthday")
+                .BuildWithParameters();
+
+            queryResult
+                .Query
+                .Trim()
+                .Should()
+                .Be("SELECT [Id],[Name],[Birthday] FROM [user] WHERE [Age] < @p0 ORDER BY [Name],[Birthday]");
+
+            dynamic parameters = queryResult.Parameters;
+
+            Assert.Equal(29, parameters.@p0);
+        }
+
+        [Fact]
+        public void ShouldCreateSelectBirthdayWithWhereAndMultipleOrderByAscDesc()
+        {
+            var queryResult = FlepperQueryBuilder.Select("Id", "Name", "Birthday")
+                .From("user")
+                .Where("Age").LessThan(29)
+                .OrderBy("Birthday", true)
```

Comment:
Instead using bollean attributes, create a new method OrderByDescending, because it is better to read and understand what is happening. So try to avoid this kind of bollean attributes